app/helpers/application_helper.rb

```diff
@@ -234,6 +234,8 @@ def action_buttons(*args)
   def avatar_image_tag(user, html_options = {})
     if user.avatar_hash.present?
       image_tag("avatars/#{user.avatar_hash}.jpg", html_options)
+    elsif user.disabled?
+      icon_text("ban #{html_options[:class]}", "", :kind => "fa", :title => "This user is disabled and won't be able to perform any actions.")
```

Comment:
nit: the title should be translated.

Reply:
Done. Thanks for the suggestions!

Reply:
Thanks :D But what I meant by "translated" is that your text string should be wrapped with `_()` function, so the translation is done automatically.